pypeit/tests/test_scripts.py

```diff
@@ -327,6 +328,17 @@ def test_obslog():
     # Clean up
     shutil.rmtree(setupdir)
 
+@dev_suite_required
+def test_compare_sky():
+    spec_file = os.path.join(os.getenv('PYPEIT_DEV'), 'Cooked', 'Science',
+                             'spec1d_b27-J1217p3905_KASTb_20150520T045733.560.fits')
+    sky_file = os.path.join(resource_filename('pypeit', 'data/sky_spec/'),
+                                              'sky_kastb_600.fits')
+
+    # Running in `test` mode
+    pargs = scripts.compare_sky.CompareSky.parse_args([spec_file, sky_file, '--test'])
+    scripts.compare_sky.CompareSky.main(pargs)
+
```

Comment:
This test should also run the script with --optimal so that it can catch changes to those keywords as well changes as to the boxcar keywords.

Reply:
Added.